release_notes.md

```
@@ -1,20 +1,30 @@
 
 ### 21Oct24
+
 - Fixed alternative world not have icon make it crash
-- Added toolbar to world when clicked
\ No newline at end of file
+- Added toolbar to world when clicked
+
+### 24Oct24
+
+- Removed shizuku needed for default folder access
+- Added setting tab with custom folder scan
```

Markdownlint

Files should end with a single newline character

(MD047, single-trailing-newline)